npm/webpack-preprocessor/index.ts

```diff
@@ -247,7 +246,11 @@ const preprocessor: WebpackPreprocessor = (options: PreprocessorOptions = {}): F
 
       debug(`errored bundling ${outputPath}`, err.message)
 
-      latestBundle.reject(err)
+      bundles[filePath].deferreds.forEach((deferred) => {
+        deferred.reject(err)
+      })
+
+      bundles[filePath].deferreds.length = 0
```

Comment:
Is there a reason to clear the array in this manner instead of `bundles[filePath].deferreds = []`?

Reply:
That would be ok too as the array is not captured - no reason except habit.